src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String[] setOfNames = new String[]{"supply", "buy"};
+    private StringBuilder dataForStatistics = new StringBuilder();
+    private List<String> fullStrings = new ArrayList<>();
```

Comment:
let's make these two variables local

Reply:
Sorry, but variable as fullStrings is used in 3 methods, that affect the final result